operator/tests/central/basic/76-switch-central-db-to-internal.yaml

```diff
@@ -5,6 +5,6 @@ metadata:
 spec:
   central:
     db:
-      isEnabled: "Enabled"
```

Comment:
This step makes no sense now, does it? There is nothing to switch, right?

Reply:
I had an impression that it's about switch from external postgres DB from `connectionString` to internal postgres central-db. if so, it makes sense because we assert that central PVC, deployment, secret is created.

Reply:
Hm, I think we need to take a look at the whole directory holistically and perhaps rename or remove some steps.

If my understanding is correct, the `10-central-cr.yaml` file contains no `spec.central.db` field therefore it should provision a `central-db` since this is the new default and only supported option in the coming release, right? In that case we should probably add the associated resources to that step's assert file to make this explicit.

In that light, the `74-central-with-db-defaults.yaml` is mostly a no-op, and I think step 74 could be dropped?

Regardless of that, I think step 75 will delete the central-db deployment and thus (in our case) result in a non-functional central because the connection string is a dummy one, correct? I'm not sure this use of switching from an already provisioned central-db to an external DB is a case worth testing - it only makes sense if the user has cloned the database into an external postgres instance out-of-band. Since we're not doing the clone here, we don't learn much from this step... I guess we could leave it around, I have no strong opinions. If we leave it around then it perhaps makes sense to add the central-db PVC to the assert file to make it explicit that we're not deleting it.

Then step 76 is fine, to make sure we can go back to using the (now orphaned after step 75) PVC. It might make sense to set the whole `db` key value to `null` though to make the test a bit more robust (in case step 74 ever sets more properties under it that might not be compatible such as size or storage class).